### PR TITLE
Vmb

### PR DESCRIPTION
**Pull request by vaishnavi7096**
PR includes:
Message Model which contains properties such as:
- senderId
- receiverId
- dateCreated
- messageId

Also changes mades in Users File and Cronjobs File

### CODE REVIEW

File: src/cronjobs [hunk 1]
```diff
@@ -1 +1,7 @@
...
+// Modified by system (1) at 2018-02-23T19:18:22Z
+p({"class":"foam.nanos.cron.Cron","minute":1,"hour":-1,"dayOfMonth":-1,"month":-1,"dayOfWeek":-1,"scheduledTime":"2018-02-23T20:01:00.0Z","id":"sessionExpiry","description":"Remove expired sessions.","lastRun":"2018-02-23T19:18:22.623Z","lastDuration":8,"server":true,"scheduled":false,"code":"dao = x.get(\"sessionDAO\");\ndao.where(foam.mlang.MLang.LT(foam.nanos.session.Session.EXPIRY, new Date())).removeAll();","output":"","notes":""})
+// Modified by system (1) at 2018-02-23T20:01:40Z
+p({"class":"foam.nanos.cron.Cron","minute":1,"hour":-1,"dayOfMonth":-1,"month":-1,"dayOfWeek":-1,"scheduledTime":"2018-02-23T21:01:00.0Z","id":"sessionExpiry","description":"Remove expired sessions.","lastRun":"2018-02-23T20:01:40.703Z","lastDuration":8,"server":true,"scheduled":false,"code":"dao = x.get(\"sessionDAO\");\ndao.where(foam.mlang.MLang.LT(foam.nanos.session.Session.EXPIRY, new Date())).removeAll();","output":"","notes":""})
```
Comment (zahnno) [inline src/cronjobs]: Remove lines 2 - 7.

File: src/users [hunk 2]
```diff
@@ -1 +0,0 @@
..."},"type":"","birthday":"1982-07-07T23:12:00.0Z","address":{"class":"foam.nanos.auth.Address","type":"","verified":false,"deleted":false,"structured":true,"address1":"","address2":"","suite":"300","city":"Toronto","postalCode":"M6G2H3","countryId":"CA","regionId":"ON","encrypted":false,"latitude":43.68018341064453,"longitude":-79.4206314086914,"streetNumber":"","streetName":"","hours":[{"class":"foam.nanos.auth.Hours","day":0,"open":true},{"class":"foam.nanos.auth.Hours","day":1,"open":true},{"class":"foam.nanos.auth.Hours","day":2,"open":true},{"class":"foam.nanos.auth.Hours","day":3,"open":true},{"class":"foam.nanos.auth.Hours","day":4,"open":true},{"class":"foam.nanos.auth.Hours","day":5,"open":true},{"class":"foam.nanos.auth.Hours","day":6,"open":true}]},"language":"en","timeZone":"","password":"SEDjby8u5eM=:ION/k78lY9RJx74MyzbWvfAtPGxoL6NRIQ6zXKZJbqHGHJ4LP6XEELayaUqDxfkOeZ8135M4pG6M+KadqqaJ/w==","previousPassword":"","note":"","businessName":"","businessIdentificationNumber":"","issuingAuthority":"","bankIdentificationCode":"","website":"","group":"system","deviceToken":"","businessTypeId":0,"businessSectorId":1,"branchId":"","clearingId":"","status":"","emailVerified":false})
```
Comment (zahnno) [inline src/users:1]: keep this user.

File: src/users [hunk 3]
```diff
@@ -1 +1 @@
...
```
Comment (zahnno) [inline src/users:1]: should be p({"class ....etc instead of -p({"class: ...etc.
Remove the "-"

File: src/files.js [hunk 4]
```diff
@@ -486,5 +486,8 @@ FOAM_FILES([
   { name: "foam/crypto/hash/Hashable" },
   { name: "foam/crypto/sign/Signer" },
   { name: "foam/crypto/sign/Signable" },
-  { name: "foam/crypto/sign/SignedFObject" }
+  { name: "foam/crypto/sign/SignedFObject" },
+
+  //Help Desk Support Files
+  { name: "foam/support/model/MessageModel"}
```
Comment (kgrgreer) [inline src/files.js:492]: We shouldn't add support models to the files.js as that will make all FOAM apps larger, even those that don't use the support features. Instead, create a new support.js file list to contain all of the support files. Like what we do for nanos.js.

File: src/foam/support/model/messageModel.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+foam.CLASS({
+    package: 'foam.support.model',
```
Comment (kgrgreer) [inline src/foam/support/model/messageModel.js:2]: Indent two spaces. See the FOAM style guide:
https://github.com/foam-framework/foam2/blob/master/doc/guides/StyleGuide.md

File: src/foam/support/model/messageModel.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    name: 'MessageModel',
+    extends: 'foam.u2.Controller',
```
Comment (kgrgreer) [inline src/foam/support/model/messageModel.js:4]: Why does this extend Controller?

**Pull request closed without merging**